packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -47,10 +48,32 @@ interface AddressProps {
     output: Partial<Output>;
 }
 
+const useLastTwoValues = () => {
+    const [lastTwoValues, setLastTwoValues] = useState<string[]>([]);
+
+    const addValue = (value: string) => {
+        let newValues;
+        setLastTwoValues(prevValues => {
+            newValues = [...prevValues, value];
+            if (newValues.length > 2) {
+                newValues.shift();
+            }
+
+            return newValues;
+        });
+
+        return newValues;
+    };
+
+    return { lastTwoValues, addValue };
+};
```

Comment:
I think this hook is not needed, info in some comment below. Moreover, it should not be part of this file

Reply:
yes, removed